pandas/core/ops/__init__.py

```diff
@@ -53,7 +56,7 @@
     kleene_xor,
 )
 from pandas.core.ops.methods import add_flex_arithmetic_methods  # noqa:F401
-from pandas.core.ops.roperator import (  # noqa:F401
+from pandas.core.roperator import (  # noqa:F401
```

Comment:
these should no longer be needed in the namespace

Reply:
We still have *lots* of places where we use  it directly from pandas.core.ops (like `ops.radd`).

Reply:
OK